invenio_search/api.py

```diff
@@ -65,8 +66,13 @@ def search(self, user_info=None, collection=None, **kwargs):
 
         for walker in search_walkers():
             query = query.accept(walker)
-        return Results(query)
 
+        index = cfg["SEARCH_ELASTIC_COLLECTION_INDEX_MAPPING"].get(
```

Comment:
Yes, I created INSPIRE mapping for all collections here https://github.com/inspirehep/inspire-next/pull/305/files#diff-5cb51431531e558f91af5de5295ed103R293

What I am not sure yet is how we do it if we want to force to look into two indices like `records-hep, records-jobs`

Not sure I understand what you mean with creating the aliases.